rts/lra/lra-test/lra-test-arquillian-extension/src/main/java/io/narayana/lra/arquillian/NarayanaLRABaseUrlProvider.java

```diff
@@ -20,16 +20,16 @@
  * 02110-1301 USA, or see the FSF site: http://www.fsf.org.
  */
 
-package io.narayana.lra.tck.arquillian;
-
-import java.lang.annotation.Annotation;
-import java.net.MalformedURLException;
-import java.net.URL;
+package io.narayana.lra.arquillian;
 
 import org.jboss.arquillian.test.api.ArquillianResource;
 import org.jboss.arquillian.test.spi.enricher.resource.ResourceProvider;
 import org.jboss.logging.Logger;
 
+import java.lang.annotation.Annotation;
```

Comment:
I think this is code formatting, if it's not required I would favour not adding as it would aid readability